HackYourFuture-CPH/fp-class19
# feat: firebase login integration

# Description

This PR is for user authentication.

Fixes # (issue)

# How to test?

1. Delete the tables in your DB and run npm run db:setup [That is because we changed the DB schema. We changed type of user id from integer to string] 
2. We changed migrations and seeds able to test. password for all the testUser seeds is **zxcvbnm**
3. Then you can test creating new user from [log-in](http://127.0.0.1:3000/log-in) page,
4. go to user-profile page
5.  Update the user profile details by clicking on the `**edit user details**` button.
6. check [here](https://console.firebase.google.com/u/0/project/class19-fp-webshop/authentication/users) and on your local database for the user created.

# Checklist

- [] This PR is ready to be merged and not breaking any other functionality
